docs/build/client_example.html

```diff
@@ -327,7 +351,16 @@ <h2>Initializing a model<a class="headerlink" href="#initializing-a-model" title
 <li><a class="reference internal" href="#usage-of-federated-model-aggregation-service">Usage of Federated Model Aggregation service</a></li>
 <li><a class="reference internal" href="#prerequisites">Prerequisites</a></li>
 <li><a class="reference internal" href="#creation-of-initial-model-weights">Creation of initial model weights</a></li>
-<li><a class="reference internal" href="#initializing-a-model">Initializing a model</a></li>
+<li><a class="reference internal" href="#initializing-a-model">Initializing a model</a><ul>
+<li><a class="reference internal" href="#dataprofiler-example">Dataprofiler Example</a><ul>
+<li><a class="reference internal" href="#running-the-example">Running the Example</a><ul>
+<li><a class="reference internal" href="#details-on-run-example-sh">Details on run_example.sh:</a></li>
+</ul>
+</li>
+</ul>
+</li>
+</ul>
+</li>
```

Comment:
changing from a PR that @ksneab7 made to main a month ago -- validated this is expected